### ISSUE TITLE
Getting 503 on all tunnels all environments, production, staging, dev

### ISSUE THREAD

**Issue by ami-descope** [lensapp/bored-agent#307]
consistently getting on all environments using latest 0.12.2 version 503 on the tunnel

`│ [2024-12-16 22:22:47.852 +0000] ERROR: [MAIN] failed to connect HTTPError: Response code 503 (Service Unavailable)                                                     `
how can i fix this

**Comment by panuhorsmalahti** [comment]
Hi, did bored-agent work before or did you just install and start to use it?

**Comment by ami-descope** [comment]
yes it was working fine and now it started crashloop on all envs with that 503, since the domain is on your end I am not sure how to debug this further

**Comment by panuhorsmalahti** [comment]
Hi, can you send an email to support@k8slens.dev with some details like the name of the Team Space so I can check on our backend's side.
